test/common/verify_3scale_UIBBT_alerts.go

```diff
@@ -0,0 +1,190 @@
+package common
+
+import (
+	"context"
+	goctx "context"
+	"fmt"
+	openshiftappsv1 "github.com/openshift/api/apps/v1"
+	k8sappsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+type Alert struct {
+	Labels struct {
+		AlertName string `json:"alertname"`
+	} `json:"labels"`
+	State string `json:"state"`
+}
+
+type AlertsResults struct {
+	Alerts []Alert `json:"alerts"`
+}
+
+func TestThreeScaleUIBBTAlerts(t TestingTB, ctx *TestingContext) {
+
+	// Get the kube client and namespace from the shared test context
+	client := ctx.Client
+
+	prefix := GetNamespacePrefix()
```

Comment:
It seems that the simplest way to get namespace prefix is to just use `NamespacePrefix` defined here https://github.com/integr8ly/integreatly-operator/blob/master/test/common/types.go#L19